src/getProjectId.js

```diff
@@ -21,10 +22,19 @@ module.exports = function(options, allowNull = false) {
 
     if (aliasCount === 0) {
       throw new FirebaseError(
-        "No project active. Run with " +
-          clc.bold("--project <projectId>") +
-          " or define an alias by\nrunning " +
-          clc.bold("firebase use --add"),
+        "No currently active project.\n" +
+          "To run this command, you have two options:\n" +
+          "- Run this command with " +
+          clc.bold("--project <project-id | project-alias>") +
```

Comment:
This should match what the current help for the `--project` flag says: `alias_or_project_id`

It probably won't stay that way, but consistency is king.

Reply:
good point by @bkendall.  We can update _all the places_ at once if we're consistent here (even if it isn't "perfect").